### ISSUE TITLE
spawn() doesn't return prototypes.

### ISSUE THREAD

**Issue by DamnedScholar** [evennia/evennia#1408]
#### Brief summary of issue:
I have a bunch of prototypes loaded in, and according to [the Spawner documentation](https://github.com/evennia/evennia/wiki/Spawner#using-evenniautilsspawner) I should be able to get a list of them, but it's not showing up. It does not seem to make a difference for this whether the prototypes are in `world/protoypes.py` or a different file.

#### Steps to reproduce the issue:
```python
>>> evennia.utils.spawner.spawn({"key": "CYBERLIMB", "location": self})
[<Object: CYBERLIMB>]
>>> evennia.utils.spawner.spawn(return_protoypes=True)
[]
```

#### Version
```
Evennia 0.6.0 (rev d9e8ab84)
```

**Comment by Griatch** [comment]
@DamnedScholar Doing a single `.spawn` call with a throwaway prototype dictionary like that will never affect what you get back from `.spawn(return_prototypes=True)` so your example behaves as expected. You *should* get back what you have defined in any of the modules defined in `settings.PROTOTYPE_MODULES' though. And testing this on `devel` branch, I get my prototypes just fine. So unless more info emerges I'm marking this to close when devel merges.

**Comment by DamnedScholar** [comment]
I don't know what you mean by "a throwaway prototype dictionary". The protoypes *are* registered in [`settings.PROTOTYPE_MODULES`](https://github.com/DamnedScholar/evennia-testbox/blob/master/server/conf/settings.py#L111).

**Comment by DamnedScholar** [comment]
Bloody Hell. Now I can't even reproduce the error. For the record, nothing has changed regarding the status of the prototypes. I was mistaken when making this issue because I typed `"key"` instead of `"prototype"`, but I was definitely seeing nothing come back when I asked for the prototypes. I just tried it again and they're all right there. So I'm closing this, but there was definitely an occurrence of the list of prototypes being inaccessible.
